src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    private static final Random random = new Random();
```

Comment:
Don't use static methods in your solution. Static methods are generally considered bad practice in object-oriented programming. Instead, you should create an instance of a class which method you want to call.